SpringProject/src/main/java/com/gradproject2019/Application.java

```diff
@@ -12,28 +12,28 @@
 @SpringBootApplication
 public class Application implements CommandLineRunner {
 
-	@Autowired
-	private DataSource dataSource;
+    @Autowired
```

Comment:
You should also choose a format for your code and stick to it, for example, 4 spaces